src/main/java/technology/tabula/Page.java

```diff
@@ -13,243 +13,271 @@
 // TODO: this class should probably be called "PageArea" or something like that
 public class Page extends Rectangle {
 
```

Comment:
I realized now that revising this change is going to be difficult. If you open both files you will notice that I just moved the methods.